appserver/tests/appserv-tests/devtests/web/servlet-4.0/defaultContextPathEar/build.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+
+    DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+    Copyright (c) 1997-2017 Oracle and/or its affiliates. All rights reserved.
+
+    The contents of this file are subject to the terms of either the GNU
+    General Public License Version 2 only ("GPL") or the Common Development
+    and Distribution License("CDDL") (collectively, the "License").  You
+    may not use this file except in compliance with the License.  You can
+    obtain a copy of the License at
+    https://glassfish.dev.java.net/public/CDDL+GPL_1_1.html
```

Comment:
Please use newer version of copyright header. e.g.
https://github.com/javaee/glassfish/blob/master/appserver/tests/appserv-tests/devtests/web/servlet-4.0/compositeTrailerSupplier/build.xml
This comment is applicable for all java and xml files